tokens/valet-core/wpvip-product-core.json

```diff
@@ -216,6 +216,28 @@
         "type": "color"
       }
     },
+    "slider": {
```

Comment:
I don't think we want this housed under `input`. Looks like it's just it's own category from the Figma file.